app/services/fixed_region_info.rb

```diff
@@ -1,53 +1,53 @@
 class FixedRegionInfo
   def call(data)
     streams_ids = streams_ids(data)
-    last_measurement_time = last_measurement_time(streams_ids)
 
      {
-      average: calculate_average(streams_ids, last_measurement_time).to_f,
+      average: calculate_average(streams_ids).to_f,
```

Comment:
What do you think of dropping the `to_f` and use `fdiv` in `calculate_average`?

https://apidock.com/ruby/Numeric/fdiv